core/SConscript.firmware

```diff
@@ -72,6 +72,7 @@ CPPDEFINES_MOD += [
     ('USE_CARDANO', '1' if EVERYTHING else '0'),
     ('USE_NEM', '1' if (EVERYTHING and TREZOR_MODEL != "R") else '0'),
     ('USE_EOS', '1' if (EVERYTHING and TREZOR_MODEL != "R") else '0'),
+    ('USE_SOLANA', '1' if EVERYTHING else '0'),
```

Comment:
this is only relevant for conditional compilation of trezor-crypto, which this PR doesn't modify. so please remove from here and sconscript.unix

Reply:
Removed https://github.com/trezor/trezor-firmware/pull/3359/commits/853af2f57fa017c5ae628344d4e1363c68936664#diff-2bd28a262d65469606d7b9a80760c5d905e26ab8fe8406704173eaac218f56b6L75.